src/Stack/Package.hs

```diff
@@ -27,56 +27,62 @@ module Stack.Package
   ,applyForceCustomBuild
   ) where
 
-import qualified Data.ByteString.Lazy.Char8 as CL8
```

Comment:
I typically ask not to apply stylish-haskell changes like this in PRs, it makes it difficult to see what has actually changed.